src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -171,15 +171,35 @@
                 <h3>Request Headers</h3>
                 {{ include('@WebProfiler/Profiler/bag.html.twig', { bag: collector.requestheaders, labels: ['Header', 'Value'], maxDepth: 1 }, with_context = false) }}
 
-                <h3>Request Content</h3>
+                <h3>
+                    Request Content
+                </h3>
 
                 {% if collector.content == false %}
                     <div class="empty">
                         <p>Request content not available (it was retrieved as a resource).</p>
                     </div>
                 {% elseif collector.content %}
-                    <div class="card">
-                        <pre class="break-long-words">{{ collector.content }}</pre>
+                    <div class="sf-tabs">
+                        {% if collector.requestheaders.get('content-type')|lower is same as ('application/json') %}
```

Comment:
Can you use a regex instead to also prettify `application/ld+json`, `application/hal+json` and so on?

Reply:
Good catch. I've used `application/([a-z]+\+)?json`, however, am open to suggestions on improving this.